webextensions/javascript-apis.json

```diff
@@ -5530,7 +5530,10 @@
                     "version_added": "26"
                   },
                   "edge": {
-                    "version_added": true
+                    "version_added": true,
+                    "notes": [
+                      "The `tab` object is incomplete."
```

Comment:
We just tested it, and it seems the `sender.tab` object is pretty complete. In fact it even shows some properties that according to [Microsoft's documentation](https://docs.microsoft.com/en-us/microsoft-edge/extensions/api-support/supported-apis#tabs) aren't even implemented yet.

![](https://i.imgur.com/G9ieJuL.png)

So I updated the pull request, and removed this note.